### DISCUSSION TITLE
LFX Mentorship Spring 2022 Project Applications

### DISCUSSION THREAD

**Discussion by nate-double-u** [cncf/mentoring#445] [Announcements]
Project applications are open for LFX Mentorship Spring'2022!

**If you are a CNCF project maintainer**, please consider proposing your project ideas to  https://github.com/cncf/mentoring/blob/main/lfx-mentorship/2022/01-Spring/project_ideas.md (by **Feb 1st**) to participate in the upcoming cycle as a mentor.

_NOTE_: This is an opportunity for the project/mentors only. **Prospective mentees** will be able to apply to participate in the program after **Feb 2nd** - https://github.com/cncf/mentoring/tree/main/lfx-mentorship/2022/01-Spring#timeline.

(as per https://cloud-native.slack.com/archives/CGPK98JNQ/p1643194862014500)

# Timeline

## Spring Term: March 1st - May 31st

* Jan 26th - Feb 1st, 2022 (1 week): Project applications open
* Feb 2nd - Feb 13th, 2022 (2 weeks): Mentee applications open
* Feb 15th - Feb 25th, 2022: Application review by the mentors/admission decisions/HR paperwork

## Mentorship duration - three months (12 weeks - full-time schedule)

* March 1, 2022 (Week 1): Mentorship program begins with the initial work assignments
* April 12, 2022 (End of Week 6): Midterm mentee evaluations and first stipend payments
* May 31, 2022 (End of Week 12): Final mentee evaluations and mentee feedback/blog submission due, second and final stipend payment approvals

**Comment by nate-double-u** [comment]
Hello project maintainers,

Just a friendly reminder to submit your project ideas by EOD today!

**Comment by idvoretskyi** [comment]
We’ve received a good number of mentoring proposals for the upcoming LFX Spring cycle - https://github.com/cncf/mentoring/blob/main/lfx-mentorship/2022/01-Spring/project_ideas.md.

Accepted projects will be pushed to the LFX website later today. Once they’re are uploaded and approved on the LFX platform side, they will be available for the mentee’s applications.

**Reply by RainbowMango** [reply]
I can see [this program](https://mentorship.lfx.linuxfoundation.org/project/6b2d49dd-fcd3-480e-838d-7310d63c5823) is waiting for approval, please let me know if there is something wrong. Thanks~~

**Reply by nate-double-u** [reply]
There is an issue with the system, we're hoping to get it sorted soon.

We'll do updates in the thread for mentees looking to apply: #478

**Reply by nate-double-u** [reply]
The system is fixed, and LFX mentee applications are now open!

**Comment by nate-double-u** [comment]
An update for mentees looking to apply: https://github.com/cncf/mentoring/discussions/478

**Comment by nate-double-u** [comment]
Hello everyone,

Here is a friendly reminder that if your project is participating in the Spring 2022 LFX Mentorship program, mentee applicant selections are due by Feb 25th, 2022.

If you've not received the _LFX Mentorship - Spring'2022 mentees acceptance form_, please let us know.

**Reply by RainbowMango** [reply]
Hi @nate-double-u Can you help check the mentee status of [this program](https://mentorship.lfx.linuxfoundation.org/project/3adb1a6d-73db-44db-8ae8-bf57367e345f)? I post my result before the deadline, but all mentees are in `Declined` state. And I also sent an email to mentoring@cncf.io.
